tests/passw_hardening/conftest.py

```diff
@@ -71,9 +71,11 @@ def clean_passw_en_dis_policies(duthosts, enum_rand_one_per_hwsku_hostname):
     duthost = duthosts[enum_rand_one_per_hwsku_hostname]
     passw_hardening_utils.config_user(duthost=duthost, username=passw_hardening_utils.USERNAME_SIMPLE_0, mode='del')
     passw_hardening_utils.config_user(duthost=duthost, username=passw_hardening_utils.USERNAME_SIMPLE_1, mode='del')
+    passw_hardening_utils.config_user(duthost=duthost, username=passw_hardening_utils.USERNAME_SIMPLE_2, mode='del')
     passw_hardening_utils.config_user(duthost=duthost, username=passw_hardening_utils.USERNAME_STRONG, mode='del')
     duthost.shell('sed -i /^'+passw_hardening_utils.USERNAME_SIMPLE_0+':/d /etc/security/opasswd')
     duthost.shell('sed -i /^'+passw_hardening_utils.USERNAME_SIMPLE_1+':/d /etc/security/opasswd')
+    duthost.shell('sed -i /^'+passw_hardening_utils.USERNAME_SIMPLE_2+':/d /etc/security/opasswd')
     duthost.shell('sed -i /^'+passw_hardening_utils.USERNAME_STRONG+':/d /etc/security/opasswd')
```

Comment:
Is `/etc/security/opasswd` actually used by `pam_pwquality`? It looks like this file is used by `pam_history`, and `pam_pwquality` can only check the previous password if it's entered on the command line, based on [this](https://github.com/libpwquality/libpwquality/issues/57).

Reply:
you test the functionaly of password hardening, the create new users with different passwords,
once you create a user in Linux, this password may be save in this file to be used later for pam_history.
The required to clean every modification in the system, so its cleaning this file as well.